Assets/_Dev/Scripts/UI/iter/iter_LevelUp.cs

```diff
@@ -78,32 +78,36 @@ public void SetData(LEVELUP_STATE lEVELUP_STATE)
     private void SetNextLevelCost()
     {
         // 다음 레벨
-        int nextLevel = default;
+        int currentLevel = default;
+        int plus = default;
         switch (lEVELUP_STATE)
         {
             case LEVELUP_STATE.WHEEL:
-                nextLevel = DBManager.instance.MyStatus.level_wheel + 1;
+                currentLevel = DBManager.instance.MyStatus.level_wheel;
                 break;
             case LEVELUP_STATE.CLICK:
-                nextLevel = DBManager.instance.MyStatus.level_click + 1;
+                currentLevel = DBManager.instance.MyStatus.level_click;
+                plus = DBManager.instance.data_Gold_Clicks.GetData(currentLevel).gold;
                 break;
             case LEVELUP_STATE.AUTO:
-                nextLevel = DBManager.instance.MyStatus.level_auto + 1;
+                currentLevel = DBManager.instance.MyStatus.level_auto;
+                plus = DBManager.instance.data_Gold_Autos.GetData(currentLevel).gold;
                 break;
             default:
                 break;
         }
+        text_Plus.text = plus == default ? default : $"(+{plus.ToString("N0")})";
```

Comment:
이런 코드는 기본적인 코드로 써 주는게 가독성이 있을 겁니다.

```suggestion
        if (plus != default)
        {
            text_Plus.text = $"(+{plus.ToString("N0")})";
        }
```